examples/hand_landmarker/ios/HandLandmarker/Views/OverlayView.swift

```diff
@@ -0,0 +1,255 @@
+// Copyright 2023 The MediaPipe Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import MediaPipeTasksVision
+
+/// A straight line.
+struct Line {
+  let from: CGPoint
+  let to: CGPoint
+}
+
+/// Line connection
+struct LineConnection {
+  let lines: [Line]
+}
+
+/**
+ This structure holds the display parameters for the overlay to be drawon on a hand landmarker object.
+ */
+struct ObjectOverlay {
+  let dots: [CGPoint]
+  let lineConnectios: [LineConnection]
```

Comment:
Do you need an array here? Also is the struct `LineConnection` needed? 
Each ObjectOverlay corresponds to one hand. So a hand overlay would need only dots and an array of lines IMO. 
LMK your thoughts.
Also would prefer `HandOverlay` instead of `ObjectOverlay`.

Reply:
@st-tuanmai Could you check my comment about the `LineConnection`?

Reply:
Sorry my bad. This is fixed.